library/core/src/num/f16.rs

```diff
@@ -1269,7 +1264,6 @@ impl f16 {
     /// ```
     #[inline]
     #[unstable(feature = "f16", issue = "116909")]
```

Comment:
Same as the f128 case, seems better to mark this `rustc_const_unstable` so that it can use the intrinsic once that is reliable.